README.md

```diff
@@ -0,0 +1,86 @@
+# TransformerQAExecutor
+
+**TransformerQAExecutor** wraps a question-answering model from huggingface. Given some questions and paragraphs/contexts, it extracts the relevant answers from the given paragraphs/contexts.
```

Comment:
```suggestion
**TransformerQAExecutor** wraps a question-answering model from Hugging Face. Given some questions and paragraphs/contexts, it extracts the relevant answers from the given paragraphs/contexts.
```

Reply:
Addressed in [this](https://github.com/jina-ai/executor-text-transformerqa/pull/7/commits/d357cee3b93a638d7189da5452209e5cc08cc1f1) commit.